_layouts/quality_estimation.html

```diff
@@ -108,9 +117,9 @@ <h3>
 </blockquote>
 
 
+{% if page.supported_languages.size > 0 %}
```

Comment:
I'd leave it how it was - this should never be 0.

If there is a bug, let it show, so we notice (and fix the bug).

If it is not a bug, let users notice.